lib/index.ts

```diff
@@ -83,7 +88,7 @@ export function maxSatisfying(
   versions: string[],
   range: string
 ): string | null {
-  return versions.reduce((x, y) => {
+  return versions.reduce<string | null>((x, y) => {
```

Comment:
why `null`? how can it happen?

Reply:
It's null when neither of the versions satisfies the range